tests/api/selfcheck/test_selfcheck.py

```diff
@@ -281,3 +282,33 @@ def test_selfcheck_checkout(client, sip2_librarian_martigny_no_email,
         )
     )
     assert res.status_code == 200
+
+
+def test_selfcheck_checkin(client, sip2_librarian_martigny_no_email,
```

Comment:
Do you have a test when an invalid item_barcode is given?

Reply:
done.